mailpoet/composer.json

```diff
@@ -18,7 +18,7 @@
     "codeception/module-cli": "^1.0",
     "codeception/module-db": "^1.1",
     "codeception/module-webdriver": "^1.4",
-    "codeception/verify": "1.5.0",
+    "codeception/verify": "^2.1",
```

Comment:
Why did you choose `2.1` and not `3.0`? https://github.com/Codeception/Verify/releases

Reply:
We declare support for PHP 7.3 in our composer.json, and version 3.0 requires PHP >= 7.4.